pages/index.vue

```diff
@@ -1,13 +1,19 @@
 <template>
   <div class="p-10">
-    <Textarea class="mb-5" placeholder="Type your message here." v-model="dotArea" />
-    <GraphView class="w-44 h-44" :dot="dot"/>
-
+    <Textarea
+      v-model="dotArea"
+      class="mb-5"
+      placeholder="Type your message here."
+    />
+    <GraphView
+      class="h-44 w-44"
+      :dot="dot"
+    />
   </div>
 </template>
 
 <script setup lang="ts">
 // https://github.com/murawakimitsuhiro/exgraph/blob/86ff28eb05b88c243ba9e0ea5b7c8962467bf14b/app.vue
-const dotArea = ref<string>("digraph {a -> c}")
-const dot = computed(() => dotArea.value)
-</script>
\ No newline at end of file
+const dotArea = ref<string>("digraph {a -> c}");
```

Comment:
This default value for dotArea changes in #1, I think it will be overwritten or at least generate a conflict.

Reply:
I think we can let this change, and fix the conflict while rebasing #1